applications/matter_weather_station/CMakeLists.txt

```diff
@@ -14,7 +14,7 @@ set(APPLICATION_CONFIG_DIR "${CMAKE_CURRENT_SOURCE_DIR}/configuration/\${BOARD}"
 set(multiprotocol_rpmsg_KCONFIG_ROOT "\\\${ZEPHYR_CONNECTEDHOMEIP_MODULE_DIR}/config/nrfconnect/chip-module/Kconfig.multiprotocol_rpmsg.root")
 set(hci_rpmsg_KCONFIG_ROOT "\\\${ZEPHYR_CONNECTEDHOMEIP_MODULE_DIR}/config/nrfconnect/chip-module/Kconfig.hci_rpmsg.root")
 
-if(OVERLAY_CONFIG STREQUAL "overlay-factory_data.conf")
+if("${OVERLAY_CONFIG}" STREQUAL "overlay-factory_data.conf")
```

Comment:
How is this a possible issue? :)

https://cmake.org/cmake/help/latest/command/if.html#strequal

Reply:
Not all of them are issues, it is the ones using variables outside of quotes that are issues, I've updated the rest so that the style throughout the code remains uniform.